packages/pages/src/generate/templates/createTemplatesJson.ts

```diff
@@ -86,9 +87,20 @@ export const getTemplatesConfig = (
     const featureConfig = convertTemplateConfigToFeatureConfig(module.config);
     features.push(featureConfig);
     const streamConfig = convertTemplateConfigToStreamConfig(module.config);
-    if (streamConfig) {
+    if (!streamConfig) {
+      continue;
+    }
+    const matchingStreamConfig = streams.find(
+      (stream) => stream.$id === streamConfig.$id
+    );
+    if (!matchingStreamConfig) {
       streams.push(streamConfig);
+      continue;
+    }
+    if (isEqual(matchingStreamConfig, streamConfig)) {
+      continue;
     }
+    throw `Conflicting configurations found for stream ID: ${streamConfig.$id}`;
```

Comment:
throw new Error('Conflicting....

Reply:
We decided to keep it like this and use logErrorAndExit so that the full stack trace isn't included.